src/provisioner/provisioner.py

```diff
@@ -398,3 +409,13 @@ def _validate_provisioning_status(cortx_conf: MappedConf, node_id: str, apply_ph
         else:
             Log.info(msg)
         return validate_result[0], validate_result[1]
+
+    @staticmethod
+    def _get_cluster_health():
```

Comment:
Semantic should be something like this:
`get_health_status(resource)`
So that same API can be used to pull status of any resource i.e cluster, pods etc...

Reply:
We need a simple query "Is Cluster Healthy". Other queries can be built into future like below.

```
CortxProvisioer: 
    def is_cluster_healthy(): 
        CortxProvisioer._get_resource_health(resource="cortx") 

    def _get_resource_health(resource): 
        # Send to HA and obtain health
```

Reply:
> def is_cluster_healthy(): 
>         CortxProvisioer._get_resource_health(resource="cortx")

Agree with this...

Reply:
done